product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Connecting to Your Cluster"
```

Comment:
IMO this page should have a link over to https://61115d4621764b0bff4fcad9--edb-docs-staging.netlify.app/docs/edbcloud/latest/administering_cluster/01_user_access/ to talk about the `edb_admin` user and password management.

Reply:
This page should also have a section on `sslmode` - we default to `require` but we should be advising customers to configure `verify-full` with a certificate in the right place, e.g., on Debian something like `&sslrootcert=/etc/ssl/certs/ca-certificates.crt`.

Reply:
@theadamwright, is this comment from Ben on your radar?

Reply:
https://github.com/EnterpriseDB/docs/pull/1713